### PR TITLE
Template sklearn xgb me

### PR DESCRIPTION
**Pull request by adi-datatron**
Updtae docker service to handle multiple exhaust file processing
1. Keeps track of processed columns are overwrite original csv file with a DF that has "processed" column
2. Add ".processed" extension if all rows are processed and skip those files in future runs
3. Fix a bug with bias and drift config (hacky fix, so please suggest cleaner way, could not find a way to easily update the args getting passed to this job)

### CODE REVIEW

**Review by TylerGillson — CHANGES_REQUESTED** [review]
See comments

File: batch_metrics.py [hunk 1]
```diff
@@ -287,15 +295,29 @@ def calculate_bias_counts(
...
+
     def process_batch(self):
         bias_counts = {"scoring_counts": {}, "feedback_counts": {}}
         drift_counts = {"continuous": {}, "categorical": {}}
+        processed_column = "processed"
 
         if not self.metric_args:
             logging.info("No metric were calculated as metric arguments were not provided.")
             return
+        self.bias_config = json.loads(self.bias_config) if isinstance(self.bias_config, str) else self.bias_config
```
[the remark this answers is not in the document]
Comment (adi-datatron) [reply batch_metrics.py:317]: Thanks!

File: batch_metrics.py [hunk 2]
```diff
@@ -414,21 +460,55 @@ def process_batch(self):
...
                         except KeyError as e:
                             logging.error(
                                 f"Could not join datatron_request_id column in either the "
                                 f"prediction file: {local_prediction_path} or "
                                 f"feedback file: {local_feedback_path} due to error: {str(e)}."
                             )
-
+                    logging.info(
+                        f"Overwriting {self.feedback_filepaths[feedback_file_count]} with {local_processed_feedback_path}")
+                    dt.authenticated_file_copy(local_processed_feedback_path,
```
[the remark this answers is not in the document]
Comment (TylerGillson) [reply batch_metrics.py:476]: Fair enough. I agree!

File: batch_metrics.py [hunk 3]
```diff
@@ -101,6 +104,9 @@ def _get_service_discovery_client():
         )
         return dsd_discovery_client
 
+    def _filter_processed_file_paths(self, paths):
```
[the remark this answers is not in the document]
Comment (adi-datatron) [reply batch_metrics.py]: Resolved!

**Review by TylerGillson — CHANGES_REQUESTED** [review]
LGTM but please add a timer for the overall batch processing as per our conversation with Jason

**Review by TylerGillson — APPROVED** [review]
LGTM

**Pull request merged**